apps/condo/domains/ai/schema/ExecutionAIFlowTask.js

```diff
@@ -0,0 +1,236 @@
+        },
+
+        result: {
+            schemaDoc: 'Result of query execution to AI.',
+            type: 'Json',
+            access: {
+                create: canOnlyServerSideWithoutUserRequest,
+                read: true,
+                update: canOnlyServerSideWithoutUserRequest,
+            },
+        },
+
+        errorMessage: {
+            schemaDoc: 'User friendly error message.',
+            type: 'Text',
+            access: {
+                create: canOnlyServerSideWithoutUserRequest,
+                read: true,
+                update: canOnlyServerSideWithoutUserRequest,
+            },
+        },
+
+        error: {
+            schemaDoc: 'Full error message for developers.',
+            type: 'Json',
+            access: {
+                create: canOnlyServerSideWithoutUserRequest,
+                read: userIsAdminOrIsSupport,
+                update: canOnlyServerSideWithoutUserRequest,
+            },
+        },
+
+        meta: {
+            schemaDoc: 'Additional arbitrary data about the task.',
+            type: 'Json',
+            access: {
+                create: canOnlyServerSideWithoutUserRequest,
+                read: userIsAdminOrIsSupport,
+                update: canOnlyServerSideWithoutUserRequest,
+            },
+        },
+
+        locale: {
+            schemaDoc: 'The requested locale in which the result will be displayed.',
+            type: 'Select',
+            options: LOCALES,
+            isRequired: true,
+            access: {
+                create: true,
+                read: true,
+                update: false,
+            },
+            hooks: {
+                resolveInput: async ({ context, resolvedData, fieldPath, operation }) => {
+                    if (operation === 'create') {
+                        const resolvedValue = resolvedData[fieldPath]
+                        if (resolvedValue && LOCALES.includes(resolvedValue)) return resolvedValue
+                        return extractReqLocale(context.req) || conf.DEFAULT_LOCALE
+                    }
+                },
+            },
+        },
+
+        user: {
+            schemaDoc: 'User that requested this operation.',
+            type: 'Relationship',
+            ref: 'User',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+            access: {
+                create: true,
+                read: true,
+                update: false,
+            },
+        },
+
+    },
+    plugins: [uuided(), versioned(), tracked(), softDeleted(), dvAndSender(), historical()],
+    access: {
+        read: access.canReadExecutionAIFlowTasks,
+        create: access.canManageExecutionAIFlowTasks,
+        update: access.canManageExecutionAIFlowTasks,
+        delete: false,
+        auth: true,
+    },
+    hooks: {
+        validateInput: (args) => {
```

Comment:
Do we need to add some kind of rate limit here? So that user will get an error immideately and task would not spawn?

Reply:
added rate limiter